core/src/main/java/io/seata/core/protocol/MergeResultMessage.java

```diff
@@ -116,7 +116,11 @@ public void decode(ByteBuffer byteBuffer) {
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder("MergeResultMessage ");
-        for (AbstractMessage msg : msgs) { sb.append(msg.toString()).append("\n"); }
+        if (msgs != null) {
```

Comment:
which sence? the msg is null ,and we need to return MergeResultMessage to client? maybe there are some bugs ?

Reply:
I simulated the transfer sence, NPE will appear when testing with multiple threads.